common/ethnicities/wc_ethnicities_elven.txt

```diff
@@ -79,30 +79,6 @@ high_elven = {
     # }
```

Comment:
Time to move to the night elves.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104633883-431f5f00-56b9-11eb-80eb-b29abd077a79.png)

</details>